modules/preview-react/pill/lib/Pill.tsx

```diff
@@ -172,11 +172,11 @@ const StyledNonInteractivePill = styled(StyledBasePill)<StyledType & CSProps>({
 });
 
 const StyledReadOnlyPill = styled(StyledNonInteractivePill)<StyledType>({
-  [buttonVars.default.background]: 'transparent',
-  [buttonVars.hover.background]: 'transparent',
-  [buttonVars.focus.background]: 'transparent',
-  [buttonVars.active.background]: 'transparent',
-  [buttonVars.disabled.background]: 'transparent',
+  [buttonColorPropVars.default.background]: 'transparent',
+  [buttonColorPropVars.hover.background]: 'transparent',
+  [buttonColorPropVars.focus.background]: 'transparent',
+  [buttonColorPropVars.active.background]: 'transparent',
+  [buttonColorPropVars.disabled.background]: 'transparent',
   border: `1px solid ${colors.licorice200}`,
```

Comment:
```suggestion
  border: `{px2rem(1)} solid ${colors.licorice200}`,
```

Also on line 46 in the same file